samples/algorithms/JointMeasurement.qs

```diff
@@ -0,0 +1,21 @@
+/// # Sample
+/// Joint Measurement
+///
+/// # Description
+/// The `Measure` operation performs a Pauli measurement. It performs a
+/// joint measurement of one or more qubits in the specified Pauli bases.
+/// The basis array and qubit array must be of the same length.
+namespace Sample {
+    open Microsoft.Quantum.Measurement;
```

Comment:
```suggestion
```

Reply:
This is not needed since `Measure` is in the `Microsoft.Quantum.Intrinsic` namespace, which is automatically opened by default.